### DISCUSSION TITLE
archives/421/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#6] [Announcements]
# archives/421/

本文首发于：https://blog.frytea.com/archives/421/ gnutls_handshake() failed: The TLS connection was non-properly terminated. 最近为新配置的虚机拉取库，但是从 GitHub 拉取

https://www.frytea.com/archives/421/

**Comment by waluuuu** [comment]
感谢大佬，github下载的问题解决了
